### DISCUSSION TITLE
Minimum/maximum temperature in thermostat card

### DISCUSSION THREAD

**Discussion by OToussaint** [home-assistant/frontend#19353] [Dashboard] [answered]
Hello there,

My boiler is only accepting a temperature between 7°C and 30°C.
My AC only between 18°C and 30°C (as defined by pyairstageAC.py).

But for the boiler, in the card, I can range from 7°C to 35°C which reset the temperature back to what it was if I overshoot the 30°C (with an error message at the bottom of the screen)
The same for the AC but there, the range available is 16°C to 32°C which is also generating an error if lower than 18°C.

Would/could it be nice to have the possibility in the card to define an optional minimum and a maximum for the gauge?

Thank you!

EDIT: Could be useful also for people who wants to deliberately restreint the range.

**Comment by karwosts** [comment, marked as answer]
I'm not sure why your integration is providing the wrong `min_temp`/`max_temp` attributes for the `climate`, but I would assume you can use the customize flow to restrict the min/max temp to whatever you desire. 

https://www.home-assistant.io/docs/configuration/customizing-devices/

This doesn't sastisfy your request for making the individual card configurable, but it should at least alleviate some of your pain points.

**Reply by OToussaint** [reply]
That's already something, I learn new things every day :) 
I thought that the customization was limited to the domains mentioned in the help page and the attributes listed there as well. Thank you!
